### ISSUE TITLE
connect through orbot button

### ISSUE THREAD

**Issue by sergdeus** [AgoraDesk-LocalMonero/agoradesk-app-foss#208]
It doesn't seem to be working if you set the proxy to 127.0.0.1:9050 (which is where orbot is listening for connections) it returns an error

Also, apparently you can use android inter-app communication to launch/connect to Orbot from inside the app, have you checked the conversations.im app code for this? https://github.com/iNPUTmice/Conversations/blob/73dc65308ff3de78d68caa7bff6715c3046c89eb/src/main/java/eu/siacs/conversations/utils/TorServiceUtils.java

Basically they send a LAUNCH_INTENT and then set the app to connect through the proxy that starts running at 127.0.0.1:9050
So a "connect through orbot" button that automates this would be great. Also setting the API domain to be the agoradesk onion domain instead of the clearnet domain
